app/src/main/java/de/danoeh/antennapod/adapter/ChaptersListAdapter.java

```diff
@@ -143,8 +143,15 @@ public View getView(final int position, View convertView, @NonNull ViewGroup par
 
         Chapter current = ChapterUtils.getCurrentChapter(media);
         if (current == sc) {
-            boolean darkTheme = UserPreferences.getTheme() == R.style.Theme_AntennaPod_Dark;
-            int highlight = darkTheme ? R.color.highlight_dark : R.color.highlight_light;
+            int theme = UserPreferences.getTheme();
+            int highlight = R.color.highlight_light;
+            if (theme == R.style.Theme_AntennaPod_Dark) {
+                highlight = R.color.highlight_dark;
+            }else if (theme == R.style.Theme_AntennaPod_TrueBlack){
```

Comment:
You are still not following a consistent code style. We are using the style as published by Google: https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Make sure you are using whitespaces in those places:

```
} else if (theme == R.style.Theme_AntennaPod_TrueBlack) {
 ^       ^                                             ^
```